maskit/datasets/__init__.py

```diff
@@ -13,6 +14,7 @@ def load_data(
     shuffle: bool = True,
     classes: Tuple[int, ...] = (6, 9),
     wires: int = 4,
+    target_length: int = 16,
```

Comment:
If this can be (usually) calculated from the others, consider to also accept ``None`` (as a default as well) and calculate it then.